runtime/compiler/control/JITClientCompilationThread.cpp

```diff
@@ -1528,25 +1528,6 @@ handleServerMessage(JITServer::ClientStream *client, TR_J9VM *fe, JITServer::Mes
          client->write(response, method->getUnresolvedFieldInCP(cpIndex));
          }
          break;
-      case MessageType::ResolvedMethod_getRemoteROMString:
-         {
-         auto recv = client->getRecvData<TR_ResolvedJ9Method *, size_t, std::string>();
-         TR_ResolvedJ9Method *method = std::get<0>(recv);
-         size_t offsetFromROMClass = std::get<1>(recv);
-         std::string offsetsStr = std::get<2>(recv);
```

Comment:
Let's use a reference here in the `MessageType::ClassInfo_getRemoteROMString:` case